### PR TITLE
Adding a property for Point's coordinates

### PR DESCRIPTION
**Pull request by Jaime02**
From my point of view, there should be a method for getting the coordinates of a point. Currently, the way I do this is using something like this:
`(point.x, point.y, point.z)`
Which would be equal to:
`point.coordinates`

#### Brief description of what is fixed or changed
Added new method for point's coordinates

#### Release Notes

* geometry
    * Added new property for getting point's coordinates

### CODE REVIEW

**Comment by smichr** [comment]
`Point(x,y,z).args` gives you the tuple containing the coordinate, `(x, y, z)`. In the SymPy context that seems sufficient.

**Comment by Jaime02** [comment]
> `Point(x,y,z).args` gives you the tuple containing the coordinate, `(x, y, z)`. In the SymPy context that seems sufficient.

Well... this could be more explicit. I don't know if that is enough.

File: sympy/geometry/point.py [hunk 1]
```diff
@@ -1029,6 +1029,21 @@ def translate(self, x=0, y=0):
...
+
+        Examples
+        ========
+
+        >>> from sympy import Point2D
+        >>> p = Point2D(0, 1)
+        >>> p.coordinates
+        (0, 1)
+        """
+        return self.args[0], self.args[1]
```
Comment (oscarbenjamin) [inline sympy/geometry/point.py]: This could just be `return self.args`

**Comment by oscarbenjamin** [comment]
I think that adding a meaningful property name for the coordinates is reasonable. Accessing args directly from outside the class makes it difficult to change the internal layout and means that all subclass have to the same args.

This will need tests though.

**Comment by smichr** [comment]
Looks good. Thanks.

**Pull request merged**